packages/wallets/src/components/offers/OfferEditor.tsx

```diff
@@ -81,6 +83,7 @@ function OfferEditor(props: OfferEditorProps) {
     let missingAssetSelection = false;
     let missingAmount = false;
     let amountExceedsSpendableBalance = false;
+    let feeInMojos = Number.parseFloat(chiaToMojo(formData.fee || 0));
```

Comment:
you can remove Number.parseFloat because chiaToMojo will return type number. 
second comment use ?? 0 instead of || 0.